apps/vendors/views.py

```
@@ -0,0 +1,91 @@
+"""Views for vendor management."""
+from typing import ClassVar, List, Type
```

Ruff (0.14.0)

`typing.List` is deprecated, use `list` instead

(UP035)